lib/site_prism/page.rb

```diff
@@ -125,7 +121,8 @@ def template_backed_matches
       matcher_template.mappings(page.current_url)
     end
 
-    def url_matches?(expected_mappings = {})
+    def url_matches?(*args)
+      expected_mappings = args.last.is_a?(::Hash) ? args.pop : {}
```

Comment:
This feels like a duplication.

If this is the only usage, then lets dry up the other call, otherwise, if you're not sure, remove this and convert the PR into reviewable.